packages/lime-smart-wifi/src/lime-smart-wifi.lua

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/lua
+
+local config = require("lime.config")
+local wireless = require("lime.wireless")
+local utils = require("lime.utils")
+
+local all_radios = wireless.scandevices()
+
+smart = {}
+
+smart.channels = {}
+smart.channels["2ghz"] = config.get("wifi", "channels_2ghz", { 1, 11, 6 })
+smart.channels["5ghz"] = config.get("wifi", "channels_5ghz", { 36, 48, 149 })
```

Comment:
Should be based on country code, you can get this information (available channels) from iwinfo library